src/main/java/org/cyclonedx/util/TimestampUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file is part of CycloneDX Core (Java).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) OWASP Foundation. All Rights Reserved.
+ */
+package org.cyclonedx.util;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@SuppressWarnings("unused")
+public final class TimestampUtils {
+  private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssX");
```

Comment:
> I would rather not introduce this for some serializers only, 

So this was already there for `MetadataDeserializer` [here](https://github.com/CycloneDX/cyclonedx-core-java/blob/eb0732750553b5bc61472131e5db89e29591ead4/src/main/java/org/cyclonedx/util/deserializer/MetadataDeserializer.java#L32). Rather than duplicating the `SimpleDateFormat` in `VulnerabilityDeserializer` I thought I would put it inside a util class. Basically similar to this [suggestion](https://github.com/CycloneDX/cyclonedx-core-java/pull/438#discussion_r1757489839).